components/elements/LabPolicyModal.tsx

```diff
@@ -0,0 +1,119 @@
+                        </div>
+                        {/* <!--Form Field End --> */}
+                        {/* <!--Form Field Start--> */}
+                        <div className="mb-6">
+                            <label htmlFor="connection-type" className="block mb-2 text-sm font-medium text-gray-900 dark:text-white">Enabled connection types</label>
+                            <select name="connection-type" id="connection-type" className="bg-gray-50 border border-gray-300 text-gray-900 text-sm rounded-lg focus:ring-blue-500 focus:border-blue-500 block w-full p-2.5 dark:bg-gray-700 dark:border-gray-600 dark:placeholder-gray-400 dark:text-white dark:focus:ring-blue-500 dark:focus:border-blue-500" required>
+                                <option value={"client connection (SSH)"}>Client Connection (SSH)</option>
+                                <option value={"server connection (SSL)"}>Server Connection (SSL)</option>
+                            </select>
+                        </div>
+                        {/* <!--Form Field End--> */}
+                        {/* <!--Form Field Start--> */}
+                        <div className="pb-6">
+                            <label htmlFor="shutdown&disconnect" className="block mb-2 text-sm font-medium text-gray-900 dark:text-white">Automatic Shutdown & Disconnect</label>
+                            <div className="flex items-center h-5 pb-4 pt-2">
+                                <input type="checkbox" id="idle-machine" value={""} className="w-4 h-4 border border-gray-300 rounded bg-gray-50 focus:ring-3 focus:ring-blue-300 dark:bg-gray-700 dark:border-gray-600 dark:focus:ring-blue-600 dark:ring-offset-gray-800"></input>
+                                <label htmlFor="idle-machine" className="block ml-2 text-sm font-medium text-gray-900 dark:text-white">Shutdown idle vitual machines</label>
+                            </div>
+                            <div className="flex items-center h-5 mb-6">
+                                <div className="grid mt-6 grid-cols-4">
+                                <input type="number" id="min-after-user-disconnect" className="col-span-1 bg-gray-50 border border-gray-300 text-gray-900 text-sm rounded-lg focus:ring-blue-500 focus:border-blue-500 block w-full p-2.5 dark:bg-gray-700 dark:border-gray-600 dark:placeholder-gray-400 dark:text-white dark:focus:ring-blue-500 dark:focus:border-blue-500" placeholder="10" required></input>
+                                <label htmlFor="min-after-user-disconnect" className="col-span-3 block mt-2 ml-2 text-sm font-medium text-gray-900 dark:text-white">Minutes after user disconnects</label>
+                                </div>
+                            </div>
```

Comment:
More spacing in between form elements
![image](https://github.com/projectcatena/cloudlabs-app/assets/63692048/186c66d2-2d3f-41f6-9438-2a851c4ef9e9)